### PR TITLE
feat: add google_container_registry

### PR DESCRIPTION
**Pull request by Pauluson**
#473 
`<pre>                                                                                                                                                    
  google_container_registry.my_registry                                                                                                             
  <font color="#555753">├─ Storage (standard)</font>                                                                     <font color="#555753">150</font>  <font color="#555753">GiB-months</font>      <font color="#555753">0.0260</font>       <font color="#555753">0.0053</font>        <font color="#555753">3.9000</font>  
  <font color="#555753">├─ Object adds, bucket/object list (class A)</font>                                                <font color="#555753">4</font>  <font color="#555753">10k operations</font>  <font color="#555753">0.0500</font>       <font color="#555753">0.0003</font>        <font color="#555753">0.2000</font>  
  <font color="#555753">├─ Object gets, retrieve bucket/object metadata (class B)</font>                                   <font color="#555753">2</font>  <font color="#555753">10k operations</font>  <font color="#555753">0.0040</font>        <font color="#555753">1e-05</font>        <font color="#555753">0.0080</font>  
  <font color="#555753">└─ Network egress</font>                                                                 
  <font color="#555753">   ├─ Data transfer in same continent</font>                                                     <font color="#555753">550</font>  <font color="#555753">GB</font>              <font color="#555753">0.0100</font>       <font color="#555753">0.0075</font>        <font color="#555753">5.5000</font>  
  <font color="#555753">   ├─ Data transfer to worldwide excluding Asia, Australia (first 1TB)</font>                  <font color="#555753">1,024</font>  <font color="#555753">GB</font>              <font color="#555753">0.1200</font>       <font color="#555753">0.1683</font>      <font color="#555753">122.8800</font>  
  <font color="#555753">   ├─ Data transfer to worldwide excluding Asia, Australia (next 9TB)</font>                   <font color="#555753">9,216</font>  <font color="#555753">GB</font>              <font color="#555753">0.1100</font>       <font color="#555753">1.3887</font>    <font color="#555753">1,013.7600</font>  
  <font color="#555753">   ├─ Data transfer to worldwide excluding Asia, Australia (over 10TB)</font>                  <font color="#555753">2,260</font>  <font color="#555753">GB</font>              <font color="#555753">0.0800</font>       <font color="#555753">0.2477</font>      <font color="#555753">180.8000</font>  
  <font color="#555753">   ├─ Data transfer to Asia excluding China, but including Hong Kong (first 1TB)</font>        <font color="#555753">1,024</font>  <font color="#555753">GB</font>              <font color="#555753">0.1200</font>       <font color="#555753">0.1683</font>      <font color="#555753">122.8800</font>  
  <font color="#555753">   ├─ Data transfer to Asia excluding China, but including Hong Kong (next 9TB)</font>           <font color="#555753">476</font>  <font color="#555753">GB</font>              <font color="#555753">0.1100</font>       <font color="#555753">0.0717</font>       <font color="#555753">52.3600</font>  
  <font color="#555753">   ├─ Data transfer to China excluding Hong Kong (first 1TB)</font>                               <font color="#555753">50</font>  <font color="#555753">GB</font>              <font color="#555753">0.2300</font>       <font color="#555753">0.0158</font>       <font color="#555753">11.5000</font>  
  <font color="#555753">   └─ Data transfer to Australia (first 1TB)</font>                                              <font color="#555753">250</font>  <font color="#555753">GB</font>              <font color="#555753">0.1900</font>       <font color="#555753">0.0651</font>       <font color="#555753">47.5000</font>  
  Total                                                                                                                       2.1388    1,561.2880  
                                                                                                                                                    
</pre>`

### CODE REVIEW

**Comment by Pauluson** [comment]
Delete Data retrieval, because it's skipping in Standard class

**Comment by Pauluson** [comment]
@alikhajeh1 this is ready to review

File: internal/providers/terraform/google/container_registry_test.go [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import (
+	"testing"
+
+	"github.com/infracost/infracost/internal/providers/terraform/tftest"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/testutil"
+	"github.com/shopspring/decimal"
+)
+
+func TestContainerRegistery(t *testing.T) {
```
Comment (aliscott) [inline internal/providers/terraform/google/container_registry_test.go:12]:
```suggestion
func TestContainerRegistry(t *testing.T) {
```

**Review by aliscott — APPROVED** [review]
LGTM

**Pull request merged**